src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```
@@ -1,177 +1,523 @@
+        return res[id];
+    }
+    function valueOf(p: any, id: string): number {
+        const entry = resourceEntry(p, id);
+        return (entry?.value ?? 0) as number;
+    }
+    function amountOf(p: any, id: string): number {
+        const entry = resourceEntry(p, id);
+        return (entry?.amount ?? 0) as number;
+    }
+
+    function calculateResourcePrice(usage: number, planLimit: number, planPricing: any): number {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)